docs/RELEASE_PROCESS.md

```diff
@@ -0,0 +1,52 @@
+# Release Process for n8n
```

Comment:
This looks like hallucinated AI slop. n8n's release process is for most part automated via github actions and does not require **any** of these manual steps. 

What was the intention behind these PRs, and why did you not consider talking to the maintainers about if and how this needed to be documented before generating any of these PRs?